flashinfer/mla/_batch_mla/_wrapper.py

```
@@ -0,0 +1,1376 @@
+
+        result = _auto_policy.plan_auto_backend(
+            plan_args,
+            candidates=tuple(auto_candidates),
+            backend_types=_BATCH_MLA_BACKENDS,
+            autotune_mode=_get_autotune_context_mode(),
+        )
+        # Publish only after selection and the real winner plan have both
+        # completed successfully. A failed replan leaves prior state live.
+        self._publish_auto_plan(plan_args, result)
+
+    # Output-only form –– ``return_lse=False`` returns the output tensor.
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)